lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Wallets.hs

```diff
@@ -998,23 +999,19 @@ spec = describe "SHELLEY_WALLETS" $ do
             matrix =
```

Comment:
Idea for integration test:
 - create empty shelley wallet that has {walletId}
 - let paymentKey = `GET /wallets/{walletId}/keys/utxo_external/0`
 - let stakeKey = `GET /wallets/{walletId}/keys/mutable_account/0`
 - Construct address:
```
POST /addresses
{payment: ${paymentKey}, stake: ${stakeKey}}
```
- verify that this address is first address on {walletId}

Reply:
Sounds like it would work :+1: